pattoo_shared/installation/packages.py

```diff
@@ -18,7 +18,11 @@ def get_package_version(package_name):
         package_dict:A dictionary containing the installed packages
 
     """
-    raw_description = shared.run_script('pip3 show {}'.format(package_name))[1]
+    try:
+        raw_description = shared.run_script('pip3 show {}'.format(package_name))[1]
+    except SystemExit:
+        message = 'The package {} is not installed'.format(package_name)
```

Comment:
Wouldn't it be better to return `None` if the package isn't installed? Then based on that value you can install the package with another function. We don't want installation to die if packages are not installed, this was one of the original issues with installation.